test/recipe/Baked.Test.Recipe.Service.Application/ConfigurationOverrider/ConfigurationOverriderFeature.cs

```diff
@@ -168,7 +210,11 @@ public void Configure(LayerConfigurator configurator)
                         HeaderItem("/", icon: "pi pi-home"),
                         HeaderItem("/report", icon: "pi pi-file", title: "Report"),
                         HeaderItem("/specs", icon: "pi pi-list-check", title: "Specs"),
-                        .. specs.Select(spec => HeaderItem($"/specs/{spec.Title.Kebaberize()}", title: spec.Title, parentRoute: "/specs"))
+                        .. specs.SelectMany(section =>
+                            section.Links.Select(link =>
+                                HeaderItem($"/specs/{section.Name.ToLower()}/{link.Title.Kebaberize()}", title: link.Title, parentRoute: "/specs")
```

Comment:
remove `section.Name` from path, page is not at that route. this causes header to disappear in spec pages